lib/cfa/placer.rb

```diff
@@ -1,6 +1,20 @@
 module CFA
-  # allows to place element at the end of configuration. Default one.
-  class AppendPlacer
+  # Places a new {AugeasElement} into an {AugeasTree}.
+  # Children of this abstract class implement different ways **where**
+  # to place the entry.
+  class Placer
+    # @param  [AugeasTree] tree
+    # @return [AugeasElement,Hash] the new element; it is empty!
+    #   Note that the return value is actually a Hash; {AugeasElement}
+    #   documents its structure.
+    def new_element(tree)
+      abstract_method(tree)
```

Comment:
Not having placer class? Maybe just document requirements for class implementing it? you can have as parameter something like `# @param [.new_element] placer getting new element that is filled with values` ? As we do not require Placer child, but any method having this method